src/py/esse/utils.py

```diff
@@ -36,8 +38,10 @@ def parseIncludeReferenceStatementsByDir(dir_path):
     data = []
     for root, dirs, files in os.walk(dir_path):
         for file_ in files:
-            file_path = os.path.join(root, file_)
-            data.append(parseIncludeReferenceStatements(file_path))
+            if not file_.startswith('.'):
+                file_path = os.path.join(root, file_)
+                data.append(parseIncludeReferenceStatements(file_path))
+    print(data)
```

Comment:
Not sure what happened in this file - can you explain, please? Inline comments can be helpful also.

Reply:
Yes sorry, i was getting an error with the `run-tests.sh` script. I wanted to check which file was the last one being ran through the script which is why i have have print lines. The program was having trouble reading the .DS_store file inside. I was getting the following error: `python 3 UnicodeDecodeError: 'utf-8' codec can't decode byte 0x86 in position 23: invalid start byte`. So i skipped over that file. by using this `if not` statement.

Reply:
Here is the full error when the code trys to look at the .DS_store file
```Traceback (most recent call last):
  File "/Users/exabyte/exabyte-stack/lib/esse/src/py/esse/tests/validate.py", line 4, in <module>
    from esse import ESSE
  File "/Users/exabyte/exabyte-stack/lib/esse/src/py/esse/__init__.py", line 6, in <module>
    SCHEMAS = parseIncludeReferenceStatementsByDir(SCHEMAS_DIR)
  File "/Users/exabyte/exabyte-stack/lib/esse/src/py/esse/utils.py", line 41, in parseIncludeReferenceStatementsByDir
    data.append(parseIncludeReferenceStatements(file_path))
  File "/Users/exabyte/exabyte-stack/lib/esse/src/py/esse/utils.py", line 23, in parseIncludeReferenceStatements
    return json.loads(json_include.build_json(dirName, baseName))
  File "/Users/exabyte/exabyte-stack/lib/esse/venv/lib/python3.8/site-packages/json_include.py", line 238, in build_json
    return JSONInclude().build_json_include(dirpath, filename, indent)
  File "/Users/exabyte/exabyte-stack/lib/esse/venv/lib/python3.8/site-packages/json_include.py", line 159, in build_json_include
    d = self._parse_json_include(dirpath, filename)
  File "/Users/exabyte/exabyte-stack/lib/esse/venv/lib/python3.8/site-packages/json_include.py", line 140, in _parse_json_include
    json_str = self._read_file(filepath)
  File "/Users/exabyte/exabyte-stack/lib/esse/venv/lib/python3.8/site-packages/json_include.py", line 32, in _read_file
    return f.read()
  File "/Users/exabyte/exabyte-stack/lib/esse/venv/lib/python3.8/codecs.py", line 322, in decode
    (result, consumed) = self._buffer_decode(data, self.errors, final)
UnicodeDecodeError: 'utf-8' codec can't decode byte 0x86 in position 23: invalid start byte
```